examples/aws-cloudwatch/capacity_planning/ebs.juttle

```diff
@@ -7,8 +7,7 @@
 // want to consider adding more volumes or faster volumes to spread
 // the work across more items.
 
-(read cloudwatch -period 3600 -last :1h: product='EBS'
- | filter name='VolumeReadOps' OR name='VolumeWriteOps'
+(read cloudwatch -period 3600 -last :1h: metric="EBS:VolumeReadOps" OR metric="EBS:VolumeWriteOps"
```

Comment:
Yeah metric="EBS:VolumeReadOps" is shorthand for "product="EBS" AND metric="VolumeReadOps". You can write it either way but the shorthand is a bit more compact.